src/kv/RocksDBStore.cc

```diff
@@ -1085,31 +1085,32 @@ int RocksDBStore::get(
     const std::set<string> &keys,
     std::map<string, bufferlist> *out)
 {
+  rocksdb::PinnableSlice value;
   utime_t start = ceph_clock_now();
   auto cf = get_cf_handle(prefix);
   if (cf) {
     for (auto& key : keys) {
-      std::string value;
+      value.Reset();
```

Comment:
just a nit, would be better if we can call Reset() after using the internal buffer. i.e., move this line after ```(*out)[key].append(value.data(), value.size());```.

Reply:
done